roles/backup/defaults/main.yml

```diff
@@ -12,6 +12,8 @@ backup_storage_requirements: ''
 # Specify storage class to determine how to dynamically create PVC's with
 backup_storage_class: ''
 
+postgres_version: 13
```

Comment:
I tested locally and doing it didn't work for me, what worked for me was removing it from the label selector below